operation/fhir-operation-everything/src/test/java/com/ibm/fhir/operation/everything/EverythingOperationTest.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0 
+ */
+package com.ibm.fhir.operation.everything;
+
+import static org.testng.AssertJUnit.assertTrue;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.AssertJUnit.assertEquals;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.List;
+
+import javax.ws.rs.core.MultivaluedMap;
+
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.model.format.Format;
+import com.ibm.fhir.model.parser.FHIRParser;
+import com.ibm.fhir.model.parser.exception.FHIRParserException;
+import com.ibm.fhir.model.resource.OperationDefinition;
+import com.ibm.fhir.model.resource.Parameters;
+import com.ibm.fhir.search.SearchConstants;
+import com.ibm.fhir.search.exception.FHIRSearchException;
+
+/**
+ * 
+ * @author Luis A. García
+ */
+public class EverythingOperationTest {
+
+    
+    private EverythingOperation exportOperation;
```

Comment:
Confusing having the operation named exportOperation
```suggestion
    private EverythingOperation everythingOperation;
```

Reply:
done